source/sync/app-builder/local-to-sync.txt

```diff
@@ -50,31 +46,31 @@ Enable Authentication
 
 Atlas Device Sync requires authenticated users. Atlas App Services provides
```

Comment:
Because we used the full name earlier, we don't need it here anymore.
```suggestion
Device Sync requires authenticated users. Atlas App Services provides
```